packages/cfd/src/Constants/jurisdiction-contents/jurisdiction-verification-contents.ts

```diff
@@ -29,7 +29,7 @@ export const jurisdictionVerificationContents = (): TJurisdictionVerificationCon
             ),
         },
         not_applicable: {
-            icon: 'IcNotApplicableVerification',
```

Comment:
can we remove whole not_applicable as it is empty string moreover in props it is optional ?
Moreover no need to tweak the test cases. As we are already mocking it there
cc: @amina-deriv @shontzu-deriv 
<img width="1263" alt="image" src="https://github.com/binary-com/deriv-app/assets/120543468/130cbe51-7f8d-40c3-a312-c8aae7bad09c">
<img width="1188" alt="image" src="https://github.com/binary-com/deriv-app/assets/120543468/f033f686-12f1-4a6f-814d-5582180ca042">

Reply:
that would work as well, but it means the `not_applicable` status will not be used and could be removed from other files i.e.:
<img width="407" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/6b9d16e8-ae1b-4792-8dfd-b97fc043030c">
<img width="417" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/6d20cc57-305d-4be7-8817-b61d2fe39133">

Reply:
I could implement your change suggestion while still keeping the `not_applicable` status from other files 👍🏼

Reply:
or I could implement your change suggestion while keeping the `not_applicable` status in case it will be useful, but as for now, its only purpose is for the (-) icon, without the icon this status is not needed

Reply:
Yes for now do we need the not_applicable status as seeing the code implementation it is just for icon and text.
Correct me if i am wrong @amina-deriv

Reply:
if we are removing 'not_applicable" i think we need more changes to the code

Reply:
i think `not_applicable` is more readable than giving as` ''`

Reply:
I prefer to keep the `not_applicable` status too, since I think it is more useful than having an `undefined` or `''`

Reply:
I prefer to keep the `not_applicable` status too, since I think it is more useful than having an `undefined` or `['']`